modules/pytket-qiskit/tests/backend_test.py

```diff
@@ -1036,3 +1036,18 @@ def test_postprocess_emu() -> None:
     r = b.get_result(h)
     shots = r.get_shots()
     assert len(shots) == 10
+
+
+@pytest.mark.timeout(None)
```

Comment:
When I run the test I see:
```
backend_test.py:1041
  /home/alec/r/pytket-extensions/modules/pytket-qiskit/tests/backend_test.py:1041: PytestUnknownMarkWarning: Unknown pytest.mark.timeout - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.timeout(None)
```
And the same for line 651.